tests/php/PHPCodeSnifferBearTest.py

```diff
@@ -39,8 +39,34 @@
 """
 
 
+bad_file_with_more_max_line_length = """<?php
+/**
+ * PHP_CodeSniffer tokenizes PHP code and detects violations of a defined \
+set of coding.This is too to test whether the absoluteLineLength.
```

Comment:
not important, but the sentence kinda ends abruptly :P (just noting :P)

Reply:
fixed